### PR TITLE
Tweaks to the Exposure Notifications

### PR DESCRIPTION
**Pull request by mattleibow**

### CODE REVIEW

File: Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs [hunk 1]
```diff
@@ -1,45 +1,42 @@
...
+		public Task ResetAllDataAsync()
 			=> NativeResetAllData().CastTask();
 
-		public Task ResetTemporaryExposureKey()
+		public Task ResetTemporaryExposureKeyAsync()
 			=> NativeResetTemporaryExposureKey().CastTask();
 
-		public Task<JavaList<TemporaryExposureKey>> GetTemporaryExposureKeyHistory()
-			=> NativeStop().CastTask<JavaList<TemporaryExposureKey>>();
+		public async Task<IList<TemporaryExposureKey>> GetTemporaryExposureKeyHistoryAsync()
```
Comment (mattleibow) [inline Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs:38]: Removed the `JavaList<T>`. Nobody likes that. Anyway, interfaces use interfaces. I think so at least... :)

File: Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs [hunk 2]
```diff
@@ -1,45 +1,42 @@
...
 
-		public Task<Java.Lang.Integer> GetMaxDiagnosisKeyCount()
-			=> NativeGetMaxDiagnosisKeyCount().CastTask<Java.Lang.Integer>();
+		public async Task<int> GetMaxDiagnosisKeyCountAsync()
+			=> (int)await NativeGetMaxDiagnosisKeyCount().CastTask<Java.Lang.Integer>();
 
-		public Task<Java.Lang.Boolean> IsEnabled()
-			=> NativeIsEnabled().CastTask<Java.Lang.Boolean>();
+		public async Task<bool> IsEnabledAsync()
+			=> (bool)await NativeIsEnabled().CastTask<Java.Lang.Boolean>();
```
Comment (mattleibow) [inline Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs:27]: Fixing those pesky casts...

File: Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs [hunk 3]
```diff
@@ -1,45 +1,42 @@
...
-using Android.Hardware;
-using Android.Runtime;
 using AndroidTask = Android.Gms.Tasks.Task;
 
 namespace Android.Gms.Nearby.ExposureNotification
 {
 	public partial interface IExposureNotificationClient
 	{
-		public Task Start(ExposureConfiguration config)
+		public Task StartAsync(ExposureConfiguration config)
```
Comment (mattleibow) [inline Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs:11]: Added the Async suffix... Is this normal? Or should it go?

Comment (Redth) [reply Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs:11]: I'm not fussed either way.

File: Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs (hunk 1 above)
Comment (Redth) [reply Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs:38]: Not sure, will the cast work properly for this? I kind of err'd on the side of not and getting a version out.

Comment (mattleibow) [reply Android/ExposureNotification/source/PlayServicesNearby/Additions/ExposureNotificationClient.cs:38]: I cast first to JavaList. This implements IList, so this is not actually a cast.

**Pull request merged**